mock/mock.go

```diff
@@ -281,15 +281,17 @@ func (m *Mock) findExpectedCall(method string, arguments ...interface{}) (int, *
 
 func (m *Mock) findClosestCall(method string, arguments ...interface{}) (*Call, string) {
 	var diffCount int
+	var repeatability int
 	var closestCall *Call
 	var err string
 
 	for _, call := range m.expectedCalls() {
 		if call.Method == method {
```

Comment:
Would it make the code clearer if we refactored this `if` clause into a guard clause? i.e. 
```
if call.Method != method {
    continue
}
...
```

Reply:
I think it is clearer without the continue, especially now that the inner content contains less logic, but let me know if you still prefer the continue.

Reply:
Thank you. We prefer a simpler approach to the code, even if it means adding a few more lines sometimes 🙂